server/repos.go

```diff
@@ -172,6 +172,16 @@ func PostRepo(c *gin.Context) {
 	name := c.Params.ByName("name")
 
 	// TODO(bradrydzewski) verify repo not exists
+	link := fmt.Sprintf(
+		"%s/api/hook",
+		httputil.GetURL(c.Request),
+	)
```

Comment:
it looks like there may have been a merge conflict? The creation of the link variable is repeated above and below. The link variable at 212 is correct and includes the token where this code (above) should be probably be removed